XM_Dinner/47_Parasect.yaml

```diff
@@ -0,0 +1,22 @@
+species: Parasect
+ingamename: MushrmCurry
+displayname: Mushroom Curry
+setname: Dinner
+tags: [Dinner]
+ball: [Cherish]
+item: Big Mushroom
+ability: Storm Drain
+moves:

```

Comment:
move order, probably warrants an upgrade to giga drain and maybe another small buff like +spatk instead of +attack